cmd/release-controller/types.go

```diff
@@ -223,6 +229,19 @@ func (m VerificationStatusMap) Incomplete(required map[string]ReleaseVerificatio
 	return names, len(names) > 0
 }
 
+func (m VerificationStatusMap) retriesRemaining(jobs map[string]ReleaseVerification) ([]string, bool) {
+	var names []string
+	for name, definition := range jobs {
+		if definition.Disabled {
+			continue
+		}
+		if s, ok := m[name]; !ok || (stringSliceContains([]string{releaseVerificationStateFailed}, s.State) && s.Retries < definition.MaxRetries) {
+			names = append(names, name)
+		}
+	}
+	return names, len(names) > 0
```

Comment:
remove the bool return value, it's redundant.  You should just check for len(names) > 0 in the places where this is called